src/hotel-management-app/src/app/customer/login/login.component.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
+<div class="login-dark">
+    <form [formGroup]="loginForm" (ngSubmit)="onLogin()">
+
+        <h2 class="sr-only">Login Form</h2>
+        <div class="illustration">
+            <i class="icon ion-ios-locked-outline"></i>
+        </div>
+        <div class="form-group">
+            <input
+                id = "email"
+                class="form-control"
+                type="email"
+                name="email"
+                placeholder="email"
+                formControlName="email">
+        </div>
+        <div class="form-group">
+            <input
+                class="form-control"
+                type="password"
+                name="password"
+                placeholder="Password"
+                id="password"
+                formControlName="password">
+        </div>
+        <div class="form-group">
+            <button class="btn btn-primary btn-block"
+                type="submit"
+                [disabled]="!loginForm.valid">Log In</button>
+        </div>
```

Comment:
Could you add a button here which the user could use to redirect to register page if he is not an existing user

Reply:
Resolved